linode_api4/objects/nodebalancer.py

```diff
@@ -187,3 +195,35 @@ def config_create(self, label=None, **kwargs):
 
         c = NodeBalancerConfig(self._client, result["id"], self.id, result)
         return c
+
+    def config_rebuild(self, config_id, nodes, **kwargs):
+        params = {
+            "nodes": nodes,
+        }
+        params.update(kwargs)
+
+        result = self._client.post(
+            "{}/configs/{}/rebuild".format(
+                NodeBalancer.api_endpoint, config_id
+            ),
+            model=self,
+            data=params,
+        )
+
+        if not "id" in result:
+            raise UnexpectedResponseError(
+                "Unexpected response rebuilding config!", json=result
+            )
+
+        return NodeBalancerConfig(self._client, result["id"], self.id, result)
```

Comment:
Could this be moved to the `NodeBalancerConfig` class? From there we should be able to call `self._populate` with the response from this request

Reply:
@yec-akamai Sorry for the late response! 
I think either layout should be good; I just figured it might make sense to move it to `NodeBalancerConfig` because the endpoint URL contains both a NodeBalancer ID and a config ID 👍